client/signup/main.jsx

```diff
@@ -727,10 +723,6 @@ class Signup extends Component {
 			return <P2SignupProcessingScreen signupSiteName={ this.state.signupSiteName } />;
 		}
 
-		if ( isNewsletterOrLinkInBioFlow( this.props.flowName ) ) {
```

Comment:
Here is where it was it's single usage

Reply:
Thank you @heavyweight 
Component removed :)